```
$ cargo nextest run
...
   Compiling zingo-infra-testutils v0.1.0 (/home/chona/zingo/zingolabs/infrastructure/main/testutils)
error[E0432]: unresolved import `zingolib::testvectors`
 --> testutils/tests/integration.rs:7:5
  |
7 |     testvectors::REG_O_ADDR_FROM_ABANDONART,
  |     ^^^^^^^^^^^ could not find `testvectors` in `zingolib`

```

`cargo check` works tho.. Feel like I might be missing something, it seems like it should resolve that import alright

--- a/testutils/tests/integration.rs
+++ b/testutils/tests/integration.rs
@@ -2,10 +2,9 @@ use std::path::PathBuf;
 
 use zcash_protocol::{PoolType, ShieldedProtocol};
 
-use zingolib::{
-    testutils::lightclient::{from_inputs, get_base_address},
-    testvectors::REG_O_ADDR_FROM_ABANDONART,
-};
+use testvectors::REG_O_ADDR_FROM_ABANDONART;
+
+use zingolib::testutils::lightclient::{from_inputs, get_base_address};
 
 use zingo_infra_testutils::client;
 

